stage_descriptions/geospatial-01-zt4.md

```diff
@@ -0,0 +1,31 @@
+In this stage, you'll add support for adding a single location under a key using the `GEOADD` command.
+
+### The `GEOADD` command
+The `GEOADD` command adds one or more location (with longitude, latitude, and name) to a key. If the key doesn’t exist, it is created. If the key exists, the locations are appended under that key.
+The syntax for `GEOADD` command is
+
+```
+GEOADD <key> <longitude> <latitude> <name>
+```
+
+Example usage:
+
+```
+> GEOADD places 13.361389 38.115556 "Palermo"
+(integer) 1
+```
+
+It returns the count of elements added, encoded as a RESP Integer.
+
+### Tests
+The tester will execute your program like this:
+
+```
+./your_program.sh
+```
+
+It will then send a `GEOADD` command specifying a key, random values of latitude and longitude, and also a random name for that co-ordinate.
```

Comment:
Let's include the actual command here, many users only look at the Tests section and skip the explanation sections